homeassistant/components/zwave_js/api.py

```diff
@@ -263,6 +274,53 @@ def node_removed(event: dict) -> None:
     )
 
 
+@websocket_api.require_admin  # type:ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/set_config_parameter",
+        vol.Required(ENTRY_ID): str,
+        vol.Required(NODE_ID): int,
+        vol.Required(PROPERTY): int,
+        vol.Optional(PROPERTY_KEY): int,
+        vol.Required(VALUE): int,
+    }
+)
+async def websocket_set_config_parameter(
```

Comment:
This could have been a separate PR, I think. The services can be used without the websocket command. The websocket command relies on the frontend.